src/decorator/tables/ClassTableChild.ts

```diff
@@ -11,7 +11,8 @@ export function ClassTableChild(tableName?: string, options?: TableOptions) {
             target: target,
             name: tableName,
             type: "class-table-child",
-            orderBy: options && options.orderBy ? options.orderBy : undefined
+            orderBy: options && options.orderBy ? options.orderBy : undefined,
+            skipSchemaSync: (options && options.skipSchemaSync === true) || false
```

Comment:
Ok, i can change it but ```options && options.skipSchemaSync === true```returns a boolean only if options isn't undefined. If options is undefined it returns undefined, not false